rule/utils.go

```diff
@@ -140,3 +140,87 @@ func isCallToExitFunction(pkgName, functionName string) bool {
 func newInternalFailureError(e error) []lint.Failure {
 	return []lint.Failure{lint.NewInternalFailure(e.Error())}
 }
+
+// isUpper checks if rune is simple digit
+//
+// # Note
+// we don't use [unicode.IsDigit] as it returns true for a large variety of digit that are not 0-9
```

Comment:
pkgsite hides unexported docs, so no need to use `[]`. Also, propose some rephrasing:

```suggestion
// The function doesn't use unicode.IsDigit as it returns true for a large variety of digits that are not 0-9.
```